lib/types/string.js

```diff
@@ -742,7 +742,7 @@ module.exports = Any.extend({
 internals.addressOptions = function (options) {
 
     if (!options) {
-        return options;
+        return internals.tlds;
```

Comment:
It will likely break in case tlds is false (which happens for frontend builds), this will probably also break coverage which is why I ignored this line as there's currently no proper way to deal with this in tests
```suggestion
        return internals.tlds || options;      // $lab:coverage:ignore$
```